src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -1,6 +1,6 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:p="/lib/plot">
```

Comment:
that's a global resources folder, you should use a unique folder for the resources, the convention is `/lib/<plugin-artifact-id>`
```suggestion
         xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:p="/lib/gerrit-trigger">
```

Reply:
You are right. Thank you so much for the tip. Changes applied.